packages/venia-ui/lib/components/CartPage/PriceAdjustments/__tests__/priceAdjustments.spec.js

```diff
@@ -3,7 +3,9 @@ import { createTestInstance } from '@magento/peregrine';
 
 import PriceAdjustments from '../priceAdjustments';
 
+jest.mock('../CouponCode', () => 'CouponCode');
 jest.mock('../giftCardSection', () => 'GiftCardSection');
+jest.mock('../GiftOptions', () => 'GiftOptions');
```

Comment:
Now that the children get rendered regardless of whether their section is open or not, we have to mock them all.